compiler/noirc_evaluator/src/ssa/ir/post_order.rs

```diff
@@ -170,4 +177,167 @@ mod tests {
         let block_a_id = func.entry_block();
         assert_eq!(post_order.0, [block_d_id, block_f_id, block_e_id, block_b_id, block_a_id]);
     }
+
+    /// Helper to construct a BasicBlockId with a syntax resembling the `b0`
+    /// syntax used in comments/ssa output.
+    fn b(id: u32) -> BasicBlockId {
+        BasicBlockId::test_new(id)
+    }
+
+    /// Documents the somewhat odd behavior from #9771
```

Comment:
```suggestion
    /// Documents the somewhat odd behavior from https://github.com/noir-lang/noir/issues/9771
```

Best to directly link so we can open these from the editor rather than having to search for an issue number.